include/os/freebsd/zfs/sys/zfs_znode_impl.h

```diff
@@ -180,7 +180,7 @@ extern int zfsfstype;
 
 extern int zfs_znode_parent_and_name(struct znode *zp, struct znode **dzpp,
     char *buf);
-extern void	zfs_inode_update(struct znode *);
+extern void	zfs_vm_update_size(struct znode *);
```

Comment:
Sure, I like the idea of changing the name to be something more platform agnostic.  But perhaps we should pick something a little more generic.  For example, on Linux we're updating the `i_size`, `i_blocks` and `i_mode` bits, not just size.  Plus it's not entirely clear what else might need to be updated on other platforms.

How about calling it `zfs_znode_update_vfs()` to indicate it's part of the `zfs_znode_*` set of interfaces and its purpose is to perform any platform specific vfs updates which may be needed.  I think using vfs and instead of vm would be more precise.

You could also use this opportunity to move the prototype in to the common `include/sys/zfs_znode.h` header.  There's no need to keep this in the platform headers.  Let's also remove the comment above the Linux version of this function.  Since we're going to want to keep this function for portability reasons it's really no longer relevant.

Reply:
update_"vfs" sounds fair enough to me.